src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx

```diff
@@ -609,3 +616,85 @@ export function isEventFromBrowserJavaScriptSDK(event: SentryTransactionEvent):
 // PerformanceMark: Duration is 0 as per https://developer.mozilla.org/en-US/docs/Web/API/PerformanceMark
 // PerformancePaintTiming: Duration is 0 as per https://developer.mozilla.org/en-US/docs/Web/API/PerformancePaintTiming
 export const durationlessBrowserOps = ['mark', 'paint'];
+
+export function getMeasurements(event: SentryTransactionEvent): Map<number, string[]> {
+  if (!event.measurements) {
+    return new Map();
+  }
+
+  const measurements = Object.keys(event.measurements)
+    .filter(name => name.startsWith('mark.'))
```

Comment:
Do all the measures start with `mark.`? Or is this added by the SDK?

Reply:
These are added by the SDK.

The convention is that measurements that begin with `mark.` are assumed to be timestamps for the occurrence of the measurement within the span view.

Reply:
Cool, didn't know that.